tests/integration_tests/cpu/detectors/wire_chamber_navigation.cpp

```diff
@@ -85,6 +88,7 @@ int main(int argc, char **argv) {
     test::helix_navigation<wire_chamber_t>::config cfg_hel_nav{};
     cfg_hel_nav.name("wire_chamber_helix_navigation");
     cfg_hel_nav.whiteboard(white_board);
+    cfg_hel_nav.propagation().navigation.min_mask_tolerance *= 0.9f;
```

Comment:
I guess this is what you meant by tweaking the test?

Reply:
Exactly. Apparently the additional cutoff error in the tolerance needed to be scaled down? This is the only detector where this happened though